reconcile/statuspage/integrations/maintenances.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import sys
+
+from reconcile.statuspage.atlassian import AtlassianStatusPageProvider
+from reconcile.statuspage.integration import get_binding_state, get_status_pages
+from reconcile.utils.runtime.integration import (
+    NoParams,
+    QontractReconcileIntegration,
+)
+from reconcile.utils.semver_helper import make_semver
+
+QONTRACT_INTEGRATION = "status-page-maintenances"
+QONTRACT_INTEGRATION_VERSION = make_semver(0, 1, 0)
+
+
+class StatusPageMaintenancesIntegration(QontractReconcileIntegration[NoParams]):
+    def __init__(self) -> None:
+        super().__init__(NoParams())
```

Comment:
Not needed

Reply:
```
Traceback (most recent call last):
  File "/home/mafriedm/repos/app-sre/qontract-reconcile/venv/bin/qontract-reconcile", line 8, in <module>
    sys.exit(integration())
             ^^^^^^^^^^^^^
  File "/home/mafriedm/repos/app-sre/qontract-reconcile/venv/lib64/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mafriedm/repos/app-sre/qontract-reconcile/venv/lib64/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/home/mafriedm/repos/app-sre/qontract-reconcile/venv/lib64/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mafriedm/repos/app-sre/qontract-reconcile/venv/lib64/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mafriedm/repos/app-sre/qontract-reconcile/venv/lib64/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mafriedm/repos/app-sre/qontract-reconcile/venv/lib64/python3.11/site-packages/click/decorators.py", line 33, in new_func
    return f(get_current_context(), *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mafriedm/repos/app-sre/qontract-reconcile/reconcile/cli.py", line 3196, in status_page_maintenances
    run_class_integration(StatusPageMaintenancesIntegration(), ctx.obj)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: QontractReconcileIntegration.__init__() missing 1 required positional argument: 'params'
```

Reply:
This is because you need to initialize the integration with `no params` as well, e.g. in cli.py:
```
run_class_integration(StatusPageMaintenancesIntegration(NoParams()), ctx.obj)
```

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4344/commits/13864b9805463542aa8cb4a7e235012fc7689ea8